src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+body {
```

Comment:
One of the task requirements is to reset browsers default styles, setting `margin: 0;` only on the body is not enough. You reset the styles either by setting default values on all elements 
```css
* {
  margin: 0;
}
```
or by creating a new file reset.css with for example [Meyer Reset](https://meyerweb.com/eric/tools/css/reset/) and importing it to the main one -> this way is better as in meyers reset you have complete control over what elements you modify as they are all listed instead of blindly setting values with `*`